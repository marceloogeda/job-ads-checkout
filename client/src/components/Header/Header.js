// @flow
import React from 'react';
import styled from 'styled-components';
import { Grid, Col, Row } from 'react-styled-flexboxgrid';

// Image
import logo from './gdp.png';

// Theme
import { colors } from './../../theme/palette';

// Style
const Header = styled.header`
  background-color: #fff;
  border-bottom: 9px solid ${colors.blue};
  display: flex;
  padding: 1.25rem 1rem;
  width: 100%;
`;

export default () => (
  <Header>
    <Grid>
      <Row>
        <Col lg={10}>
          <img src={logo} alt="global delivery pod" width={110} />
        </Col>
      </Row>
    </Grid>
  </Header>
);
