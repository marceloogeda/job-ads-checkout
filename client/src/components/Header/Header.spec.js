import React from 'react';
import TestRenderer from 'react-test-renderer';

import Header from './Header';

describe('# Header', () => {
  it('should render correctly', () => {
    const testRenderer = TestRenderer.create(<Header />);

    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
