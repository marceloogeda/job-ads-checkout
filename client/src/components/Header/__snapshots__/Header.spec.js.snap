// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# Header should render correctly 1`] = `
<header
  className="sc-bdVaJa ervVcD"
>
  <div
    className="Grid-iUdnzb huLnVS"
  >
    <div
      className="Row-ciskzD gWlJeB"
    >
      <div
        className="Col-jCOnJH eEidFm"
      >
        <img
          alt="global delivery pod"
          src="gdp.png"
          width={110}
        />
      </div>
    </div>
  </div>
</header>
`;
